com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/fs/ZipMountPoint.java

```diff
@@ -92,6 +92,10 @@ public boolean isFile() {
             boolean isDir = entry.isDirectory();
             return !isDir;
         }
+        @Override
+        public byte[] sha1() throws IOException {
+            return sha1(true);
```

Comment:
files can't be changed in this during the build - that's why here sha1  cache is ON by default